### ISSUE TITLE
Conflict when using with SDL2

### ISSUE THREAD

**Issue by MDreamer** [jazzycamel/ws28128-rpi#5]
Hi,

I tried to use the library with SDL2 and SDL2_mixer to play sound and it has some kind of conflict. Once the NeoPixel object is beimg created it kills the sound.

IT CAN'T RECOVER - only restarting the Pi... bugger

Cheers
Maayan

**Comment by RBEGamer** [comment]
you have to use a usb soundcard!
and you have to disable the audiodriver in the raspi-config

**Comment by MDreamer** [comment]
Thanks for the response. Where does it say I need to disable it? Just ordered one

**Comment by RBEGamer** [comment]
This is because the pwm pin which youre using for the libary and the sound out are sharing the same pwm on the chip...

**Comment by MDreamer** [comment]
Right. Thanks. I didn't dive into the code but it make sense. Thanks.

**Comment by RBEGamer** [comment]
So a usb soundcard is the best option for you and the Soundquality is much better if youre using a old generation pi
